### ISSUE TITLE
Wrong selector being used in AtkCocoaExtensions

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#7243]
```
  =================================================================
  	Managed Stacktrace:
  =================================================================
  	  at <unknown> <0xffffffff>
  	  at MonoDevelop.Components.Mac.Messaging:IntPtr_objc_msgSend <0x00064>
  	  at MonoDevelop.Components.AtkCocoaHelper.AtkCocoaMacExtensions:AddElementToTitle <0x000ba>
  	  at MonoDevelop.Ide.WelcomePage.WelcomePageSection:SetTitledWidget <0x000ca>
  	  at MonoDevelop.Ide.WelcomePage.WelcomePageRecentProjectsList:RecentFilesChanged <0x0033a>
  	  at MonoDevelop.Ide.WelcomePage.WelcomePageRecentProjectsList:.ctor <0x0022a>
  	  at MonoDevelop.Ide.WelcomePage.DefaultWelcomePage:BuildContent <0x006ea>
  	  at MonoDevelop.Ide.WelcomePage.WelcomePageWidget:.ctor <0x001f9>
  	  at MonoDevelop.Ide.WelcomePage.DefaultWelcomePage:.ctor <0x0001a>
  	  at MonoDevelop.Ide.WelcomePage.WelcomePageService:ShowWelcomePage <0x000ea>
  	  at MonoDevelop.Ide.IdeStartup:OnIdle <0x00082>
  	  at IdleProxy:Invoke <0x00019>
  	  at GLib.SourceProxy:HandlerInternal <0x00083>
  	  at GLib.SourceProxy:HandlerInternal <0x00062>
  	  at <unknown> <0xffffffff>
  	  at Gtk.Application:gtk_main <0x00056>
  	  at Gtk.Application:Run <0x00012>
  	  at MonoDevelop.Ide.IdeApp:Run <0x00022>
  	  at MonoDevelop.Ide.IdeStartup:Run <0x03402>
  	  at MonoDevelop.Ide.IdeStartup:Main <0x003d2>
  	  at MonoDevelop.Startup.MonoDevelopMain:Main <0x0001a>
  	  at <Module>:runtime_invoke_int_object <0x0010c>
  =================================================================
```

> VS bug [#802435](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/802435)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-11] Comment by **xamarinc**



Fixed in version 8.1.0.32 (master)



Pull Request #7244 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 410c2d90f8034937e197ce82392997dd9240d23b (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-12] Comment by **xamarinc**



Fixed in version 8.0.0.2527 (release-8.0)



Pull Request #7245 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 12d855d5ae5188fc36fd70bb1205f3c51fcbe1f5 (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-14] Comment by **xamarinc**



Fixed in version 8.1.0.1701 (master-vnext)



Pull Request #7245 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 12d855d5ae5188fc36fd70bb1205f3c51fcbe1f5 (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-21] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #7244 merged by: Lluis Sanchez  
 Author: mono  
 Commit: 410c2d90f8034937e197ce82392997dd9240d23b (xamarin/monodevelop)
